community/rfcs/techniques/design/20200306-weight-clustering-design.md

```diff
@@ -0,0 +1,270 @@
+# Keras Weight Clustering API Design
+
+| Status        | Pending                                       |
+:---------------|:----------------------------------------------|
+| **Author(s)** | Mohamed Nour Abouelseoud (mohamednour.abouelseoud@arm.com), Aron Virginas-Tar (aron.virginas-tar@arm.com), Anton Kachatkou (anton.kachatkou@arm.com)|
+| **Sponsor**   | Alan Chiao (alanchiao@google.com)             |
+| **Updated**   | 2020-03-16                                    |
+
+## Overview
+
+See the initial proposal [here](../proposals/20200205-weight-clustering-proposal.md) for
```

Comment:
Maybe worth summarizing here the user benefit will be, out of the box, model compression on disk.

Reply:
A note to readers also that this link is waiting on the RFC to be submitted to work - see
the initial proposal down below.